src/openrct2/localisation/StringIds.h

```diff
@@ -3899,8 +3899,10 @@ enum
 
     STR_INVALID_CLIMATE_ID = 6269,
 
+    STR_GUESTS_GETTING_STUCK_ON_RIDE = 6270,
+
     // Have to include resource strings (from scenarios and objects) for the time being now that language is partially working
-    STR_COUNT = 32768
+    STR_COUNT = 32769
```

Comment:
I don't think you're supposed to change this.

Reply:
Who would change it? Or is that the maximum amount of possible strings?